src/options.cpp

```diff
@@ -966,6 +966,10 @@ Options::Options()
     m_mensuralToMeasure.Init(false);
     this->Register(&m_mensuralToMeasure, "mensuralToMeasure", &m_general);
 
+    m_midiCue.SetInfo("MIDI playback of cue notes", "Include cue notes in MIDI output");
```

Comment:
Also move to the corresponding place

Reply:
What do you mean? It is in general before `m_midiTempoAdjustment`.

Reply:
Sorry, because your last modification only moved the option in the header I thought you forgot to move it in the implementation too. I had not noticed they were actually wrongly aligned in your first commit.